pkg/cli/client/client.go

```diff
@@ -9,25 +9,29 @@ import (
 	"os"
 	"time"
 
+	"github.com/tektoncd/results/pkg/cli/config"
 	pb "github.com/tektoncd/results/proto/v1alpha2/results_go_proto"
-	"github.com/tektoncd/results/tools/tkn-results/internal/config"
 	"golang.org/x/oauth2"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/oauth"
 	v1 "k8s.io/api/authentication/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
+
+	// Load auth plugins
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-type ClientFactory struct {
+// Factory contains the configuration for creating a k8s client.
+type Factory struct {
```

Comment:
Nice naming. Following the convention. :)

Reply:
Yeah, had to change a couple of names due to linting errors 😸